trailblazer/store/filters/analyses_filters.py

```diff
@@ -80,6 +80,13 @@ def filter_analyses_by_empty_comment(analyses: Query, **kwargs) -> Query:
     return analyses.filter(sqlalchemy.or_(Analysis.comment.is_(None), Analysis.comment == ""))
 
 
+def filter_analyses_by_delivered(analyses: Query, delivered: bool | None, **kwargs) -> Query:
+    if delivered == True:
+        return analyses.filter(Analysis.delivery != None)
+    if delivered == False:
+        return analyses.filter(Analysis.delivery is None)
```

Comment:
Just to double check - does this comparison work? I feel as if I always have to do "=="-comparisons in these filters even though pycharm complains.

Reply:
No, you are right - this was not correct. Fixed here https://github.com/Clinical-Genomics/trailblazer/pull/413/commits/3763426b539c9363ee851c2c9eb74a37dde83824